entity-test-lib/src/main/java/org/terracotta/passthrough/PassthroughEntityRef.java

```diff
@@ -119,7 +119,6 @@ public void create(C configuration) throws EntityNotProvidedException, EntityAlr
   public C reconfigure(C configuration) throws EntityException {
     // Make sure that we have a service provider.
     if (null != this.service) {
-      getWriteLock();
```

Comment:
It is probably worth adding a comment here, regarding why the write lock isn't required, if it is otherwise a common pattern.